client/src/Components/Input.js

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import TextField from "@material-ui/core/TextField";
+import { makeStyles } from "@material-ui/styles";
+import PropTypes from "prop-types";
+
+const Input = props => {
+  const {
+    id,
+    name,
+    label,
+    variant,
+    className,
+    disabled,
+    helperText,
+    onChange,
+    placeholder,
+    inputProps,
+  } = props;
+
+  const classes = useStyles();
+
+  return (
+    <TextField
+      id={id}
+      name={name}
+      label={label}
+      variant={variant}
+      className={`${classes.input} ${className}`}
+      disabled={disabled}
+      helperText={helperText || ""}
+      onChange={onChange}
+      placeholder={placeholder}
+      InputProps={inputProps}
+    />
+  );
+};
+
+const useStyles = makeStyles(() => ({
```

Comment:
To follow the convention of the rest of the Component files, let's bring these objects, like the Styles object and defaultProps object to the top of the component.

Reply:
If I move defaultProps to the top, it gives me a warning sign because the component is not initialised at that point? Should I still leave it up?

Reply:
Oooh, good point! Just the move the styles object and leave defaultProps, thanks!